src/imitation/policies/exploration_wrapper.py

```diff
@@ -30,21 +30,22 @@ def __init__(
         """Initializes the ExplorationWrapper.
 
         Args:
-            policy: The policy to randomize.
+            policy_callable: The policy callable to randomize. Preferably a policy
+                callable to match the same type of self._random_policy(obs) call.
```

Comment:
What does "match the same type" mean? It's type is a `rollout.PolicyCallable` (according to the annotation), what other type should be aligned?

Reply:
`def _random_policy(self, obs: np.ndarray) -> np.ndarray:` seems to be the same type as `PolicyCallable = Callable[[np.ndarray], np.ndarray]`. Perhaps no need to include it.